frontend/src/sass/base/_colours.scss

```diff
@@ -28,7 +28,7 @@ $grey3: #888888; // slider text
 // 136 136 136
 $grey4: #a0a0a0; // border of dropdown
 // 160 160 160
-
+$red: #ff0000;
```

Comment:
Very bright, can i sudgest a generic bright red but not 100%. Just few sudgestion http://www.colourlovers.com/colors/search?query=error+red